frontend/src/redux/reducers/User.tsx

```diff
@@ -17,3 +17,14 @@ export const getUsers = (store: any[] = [], action: any) => {
       return store;
   }
 };
+
+export const getCurrentUser = (store: any = {}, action: any) => { // TODO: temporary setup for single user
+  switch (action.type) {
+    case 'GET_USERS':
+      return action.users[0];
```

Comment:
Will this be changed in the future? It seems to currently grab the first user only. Otherwise LGTM

Reply:
Add(ed) to card, tbd; perhaps in future